scripts/dmsdk/gen_cpp.py

```diff
@@ -0,0 +1,224 @@
+# Copyright 2020-2024 The Defold Foundation
```

Comment:
Some of these files are heavily modified from the Sokol bindgen tools.
However, at this point I feel that I've modified it so far, that it has little resemblance of the original scripts.